src/org/zaproxy/zap/extension/websocket/ui/httppanel/models/StringWebSocketPanelViewModel.java

```diff
@@ -28,24 +28,29 @@
 	
 	private static final Logger LOGGER = Logger.getLogger(StringWebSocketPanelViewModel.class);
 	private boolean isErrorMessage;
-
+	private boolean isEditable = true;

```

Comment:
- `editable`.
 - It does not need to be initialised, the views are non-editable by default.